packages/next/server/web/spec-extension/response.ts

```diff
@@ -67,10 +67,10 @@ export class NextResponse extends Response {
     return this.cookie(name, '', { expires: new Date(1), path: '/', ...opts })
   }
 
-  static json(body: any) {
+  static json(body: any, init?: ResponseInit) {
     return new NextResponse(JSON.stringify(body), {
       headers: { 'content-type': 'application/json' },
-    })
+    }, init)
   }
```

Comment:
Shouldn't it be:

```ts
static json(body: any, init?: ResponseInit) {
  return new NextResponse(JSON.stringify(body), {
    ...init
    headers: {
      'content-type': 'application/json',
      ...init.headers
    }
  })
}
```

`NextResponse` only takes in two arguments.

Reply:
oh yea, my fault!